Formula/s-nail.rb

```diff
@@ -0,0 +1,75 @@
+class SNail < Formula
+  desc "Fork of Heirloom mailx."
+  homepage "https://www.sdaoden.eu/code.html"
+  url "https://www.sdaoden.eu/downloads/s-nail-14.9.3.tar.gz"
+  sha256 "9048abe94c8b732ddefcd70b9f6052da16977de76b07598790e4c763e97f911d"
+
+  # Define options
+  option "with-maximal", "Anything on, including experimental features."
```

Comment:
I think we'd only need to support the maximal configuration, and not have any options.